### PR TITLE
Add CookieListBlock to display cookie list from OneTrust

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
- Add CookieListBlock to display cookie list from OneTrust

The OneTrust script fetches and outputs content in the div with the `id`. Styling is handled in OneTrust dashboard.

Screenshot 2023-08-21 at 14 09 12

## Justify why they are needed
Needed for GDPR compliance

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3007** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3007" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3007?utm_source=stack-comment).

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/blocks/CookieList.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  return (
+    <GridLayout.Root>
+      <GridLayout.Content width={{ base: '1', md: '2/3', xxl: '1/2' }} align={'center'}>
+        <div id="ot-sdk-cookie-policy"></div>
+      </GridLayout.Content>
+    </GridLayout.Root>
+  )
+}
+
+CookieListBlock.blockName = 'cookieList'
```
[the remark this answers is not in the document]
Comment (gustaveen) [reply apps/store/src/blocks/CookieList.tsx:13]: oops, forgot to rename it, thanks :)

**Comment by gustaveen** [comment]
[Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3007) rebased this pull request as part of a merge.

**Comment by gustaveen** [comment]
@gustaveen merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3007).

**Pull request merged**